docker-compose.yml

```diff
@@ -27,3 +27,15 @@ services:
     volumes:
       - $ROOT/demo:/orderly
       - $ROOT/config/orderlyweb:/etc/orderly/web
+  mq:
+    image: redis
+    ports:
+      - "6379:6379"
```

Comment:
Do these ports need to be exposed to the host machine? Shouldn't be necessary if only used for communication between containers defined in this file.

If do need to be exposed then consider "127.0.0.1:6379:6379" otherwise they're exposed to other hosts on the network.

Reply:
Yeah, good point. I think we tend to expose more ports in general than we need to, almost just as a reminder of which are the relevant ports on each container... It can be useful in deployed networks for diagnostics I guess, unlikely to be needed in these sorts of dev scripts. Good point on keeping them local, we should probably do that more too.